Message Identity note when using Immediate Dispatch (#5584)
Particular/docs.particular.net: nservicebus/messaging/send-a-message.md

While it's usually best to let NServiceBus [handle all exceptions](/nservicebus/recoverability/), there are some scenarios where messages might need to be sent regardless of whether the message handler succeeds or not, for example, to send a reply notifying that there was a problem with processing the message.

> NOTE: Ensure that no ghost or duplicate messages are created if this code is executed more than once by assigning the same [message identifier](/nservicebus/messaging/message-identity) to messages sent via immediate dispatch.

Author: @andreasohlund how about this?

> NOTE: Ensure no issues ghost or duplicate messages are created if this code might be run more-than-once by assigning the same [message identifier](/nservicebus/messaging/message-identity) to message send via immediate dispatch.

Reviewer: ```suggestion
NOTE: Ensure that no ghost or duplicate messages are created if this code is executed more than once by assigning the same [message identifier](/nservicebus/messaging/message-identity) to messages sent via immediate dispatch.
```

Reviewer: Could we link to something that talk about ghost and duplicate messages?

Author: Ghost messages isn't a term that can be googled.. I'll try to rewrite your suggestion

> NOTE: Side effects can occur when failures happen after sending the message. The messages could be retried meaning duplicate messages are created if this code is executed more than once. Ensure that you are assigning the same [message identifier](/nservicebus/messaging/message-identity.md) to messages sent via immediate dispatch. Second, due to failures it could happen that messages are sent that contain state which is inconsistent because of failing operations like a storage modificanion that didn't occur.

Author: ```suggestion
NOTE: Side effects can occur when failures happen after sending the message. The messages could be retried meaning duplicate messages are created if this code is executed more than once. Ensure that the same [message identifier](/nservicebus/messaging/message-identity.md) gets assigned to messages sent via immediate dispatch when invoked more than once. Second, due to failures it could happen that messages are sent that contain state which is inconsistent because of failing operations like a storage modification that didn't occur.
```
